### PR TITLE
chore: annotate HierarchicalFacet for documentation

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

**Comment by vvo** [comment]
Let me know @bobylito if I did the right thing

File: src/SearchResults/index.js [hunk 1]
```diff
@@ -21,6 +21,17 @@ var generateHierarchicalTree = require('./generate-hierarchical-tree');
  * @property {object} stats undefined unless facet_stats is retrieved from algolia
  */
 
+ /**
+  * @typedef SearchResults.HierarchicalFacet
+  * @type {object}
+  * @property {string} name name of the current value given the hierarchical level, trimmed.
+  *                         If root node, you get the facet name
+  * @property {integer} count number of objets matching this hierarchical value
```
Comment (bobylito) [inline src/SearchResults/index.js]: The correct type in JS is number. You can specify the value more precisely in the comment.

**Comment by bobylito** [comment]
Captain obvious mode : the formatting is not correct (cf travis)

And a small comment in the changes.

**Comment by vvo** [comment]
ok fixed

**Comment by bobylito** [comment]
Great! :)

**Pull request merged**